projects/packages/my-jetpack/_inc/components/stats-section/create-stat-diff-text.js

```diff
@@ -0,0 +1,59 @@
+import { sprintf, __ } from '@wordpress/i18n';
+import formatNumber from '../../utils/format-number';
+import formatPercentage from '../../utils/format-percentage';
+
+/**
+ * Creates text describing a stat difference between two periods.
+ * The text mentions both the absolute difference and the percentage difference.
+ *
+ * @param {Function} countStat     - Function that accepts a number and calls _n() with that number to return the singular or plural form of the stat count.
+ *                                 E.g. countStat( 1 ) returns '%s view', countStat( 5 ) returns '%s views'
+ * @param {number}   count         - Current period count
+ * @param {number}   previousCount - Previous period count
+ * @return {string}  Formatted text describing the difference
+ */
+const createStatDiffText = ( countStat, count, previousCount ) => {
+	const diff = count - previousCount;
```

Comment:
If count or previousCount is not a number, should we return the 'no change' message here too?

Reply:
Good call! In that instance, it's likely something's going wrong or not enough data has been recorded yet, so it seems better to return an empty string.